dnf/cli/cli.py

```diff
@@ -861,6 +861,7 @@ def _parse_commands(self, opts, args):
         basecmd = opts.command[0]
         command_cls = self.cli_commands.get(basecmd)
         if command_cls is None:
+            self.optparser.print_help()
```

Comment:
Agree. On the other hand print_help() contains list of known commands/plugins which may be handy.
I'm fine with both options.